azure-pipelines.yml

```diff
@@ -8,11 +8,11 @@ jobs:
           test_title: Pylint All
           toxenv: pylint
           python: 3.7
-        Pylint-Subpackages:
-          image_name: ubuntu-16.04
-          test_title: Pylint Subpackages
-          toxenv: analysis,dicom,fileformats,labs,logfiles,coordsandscales,electronfactors,databases,utilities,workshops,toolbox,xlwings
-          python: 3.7
+        # Pylint-Subpackages:
+        #   image_name: ubuntu-16.04
+        #   test_title: Pylint Subpackages
+        #   toxenv: analysis,dicom,fileformats,labs,logfiles,coordsandscales,electronfactors,databases,utilities,workshops,toolbox,xlwings
+        #   python: 3.7
```

Comment:
These were taking too long. Will speed them up and include them back in another day.

Reply:
Yay!